docs/FormatsAndConventions.md

```diff
@@ -0,0 +1,65 @@
+# Formats and Conventions
+
+## Branch
+
+format: **`issueNumber-some-keywords-from-issue-title`**
```

Comment:
1. ~~f~~Format (also for other sections).
2. Also we may want to justify why we have this kind of convention. We used to have `Issue1234` before and moved to this convention for a reason. Also for the merge commit message (PR name already explained).